tests/subsys/zbus/async_listener/prj.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_ZTEST=y
+CONFIG_LOG=y
+CONFIG_ZBUS=y
+CONFIG_ZBUS_ASSERT_MOCK=y
+CONFIG_ZBUS_ASYNC_LISTENER=y
+CONFIG_HEAP_MEM_POOL_SIZE=512
```

Comment:
Not clear to me why do you need `CONFIG_HEAP_MEM_POOL_SIZE=512`.

Reply:
Not needed in fact. Removing that.